tests/data/eurosat/eurosat-test.txt

```diff
@@ -0,0 +1,3 @@
+AnnualCrop_1.tif
+Forest_1.tif
+
```

Comment:
The original split files don't have an empty line in them, I would remove this empty line